README.md

```diff
@@ -2,10 +2,73 @@
 
 This project was made due to the **[CodersCamp2021](https://www.coderscamp.edu.pl/)**
 
-# Project Setup
+# Getting Started with Create React App
```

Comment:
there is no info about how to install packages, also remove most of the part of this auto generated readme, just put the informations about installing, launching in dev mode (yarn start), and about the code format commend as wel